addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -607,19 +607,28 @@ def _compute_display_name(self):
                 target,
             )
 
-    def _add_lastcalls(self):
```

Comment:
You cannot change a method signature on a stable release.

Reply:
Changed :)